scripts/start.sh

```diff
@@ -2,9 +2,24 @@
 
 echo "Start script executing for admin.."
 
+# Query the fixed container agent IP address for credentials (search the AWS docs
+# for "Task IAM role" for more information about this endpoint)
+session_tokens=$(curl 169.254.170.2$AWS_CONTAINER_CREDENTIALS_RELATIVE_URI)
```

Comment:
That's a little disingenuous, we don't declare all variables as global variables just in case we might need them in another scope elsewhere in other code, so why would we do it here?

It slows down the actual script (for something which is literally unused in this case if CONTAINER_ROLE is set) and could in the future incur extra costs (as it's using extra cloud resources every time this is run).